### PR TITLE
Shadcn migration - heros

### PR DESCRIPTION
**Pull request by pettinarip**
## Description

Migrates the ContentHero and MdxHero component to TW.

### CODE REVIEW

File: src/components/Hero/ContentHero/index.tsx [hunk 1]
```diff
@@ -1,42 +1,22 @@
...
 
 const ContentHero = (props: ContentHeroProps) => {
-  const {
-    breadcrumbs,
-    heroImg,
-    buttons,
-    title,
-    description,
-    blurDataURL,
-    maxHeight,
```
Comment (pettinarip) [inline src/components/Hero/ContentHero/index.tsx:20]: Removed this prop as it was not being used and was a bit hacky to set up with tw.

File: src/components/Hero/MdxHero/index.tsx
Comment (wackerow) [inline src/components/Hero/MdxHero/index.tsx:1]: @pettinarip Are we using this component anywhere? I don't see any instances of it

**Review by wackerow — APPROVED** [review]
Looks good! Did leave a note about MdxHero though... are we using this somewhere or should we remove it?

File: src/components/Hero/MdxHero/index.tsx
Comment (pettinarip) [reply src/components/Hero/MdxHero/index.tsx:1]: We are not atm. But it is planned to be used once we implement the new DS styles on the docs and other pages I believe.

**Pull request merged**